docs/source/index.rst

```diff
@@ -8,18 +8,17 @@ SMAUG DOCUMENTATION
 SMAUG is a deep learning framework that enables end-to-end simulation of DL models on custom SoCs with a variety of hardware accelerators.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 3
```

Comment:
I feel like having extra depth in the ToC here adds too much visual clutter.

Reply:
Reverted the depth too.